File: snapshots/output-net6.0/syntax/Main/Enums.cs
  using System.Diagnostics.CodeAnalysis;

  namespace Main;
//          ^^^^ reference scip-dotnet nuget . . Main/

  [SuppressMessage("ReSharper", "all")]
  public class Enums
//             ^^^^^ definition scip-dotnet nuget . . Main/Enums#
//                   documentation ```cs\nclass Enums\n```
  {
      enum EnumWithIntValues
//         ^^^^^^^^^^^^^^^^^ definition scip-dotnet nuget . . Main/Enums#EnumWithIntValues#
//                           documentation ```cs\nenum EnumWithIntValues\n```
      {
          Ten = 10,
//        ^^^ definition scip-dotnet nuget . . Main/Enums#EnumWithIntValues#Ten.
//            documentation ```cs\nEnumWithIntValues.Ten\n```
          Twenty = 20
//        ^^^^^^ definition scip-dotnet nuget . . Main/Enums#EnumWithIntValues#Twenty.
//               documentation ```cs\nEnumWithIntValues.Twenty\n```
      }

      enum EnumWithByteValues
//         ^^^^^^^^^^^^^^^^^^ definition scip-dotnet nuget . . Main/Enums#EnumWithByteValues#
//                            documentation ```cs\nenum EnumWithByteValues\n```
      {
          Five = 0x05,
//        ^^^^ definition scip-dotnet nuget . . Main/Enums#EnumWithByteValues#Five.
//             documentation ```cs\nEnumWithByteValues.Five\n```
          Fifteen = 0x0F
//        ^^^^^^^ definition scip-dotnet nuget . . Main/Enums#EnumWithByteValues#Fifteen.
//                documentation ```cs\nEnumWithByteValues.Fifteen\n```
      }
  }

File: snapshots/output-net6.0/syntax/VBMain/Enums.vb
  Imports System.Diagnostics.CodeAnalysis

  Namespace VBMain
'           ^^^^^^ reference scip-dotnet nuget . . VBMain/
      <SuppressMessage("ReSharper", "all")>
      Public Class Enums
'                  ^^^^^ definition scip-dotnet nuget . . VBMain/Enums#
'                        documentation ```vb\nClass Enums\n```
          Enum EnumWithIntValues
'              ^^^^^^^^^^^^^^^^^ definition scip-dotnet nuget . . VBMain/Enums#EnumWithIntValues#
'                                documentation ```vb\nEnum EnumWithIntValues\n```
              Ten = 10
'             ^^^ definition scip-dotnet nuget . . VBMain/Enums#EnumWithIntValues#Ten.
'                 documentation ```vb\nEnumWithIntValues.Ten = 10\n```
              Twenty = 20
'             ^^^^^^ definition scip-dotnet nuget . . VBMain/Enums#EnumWithIntValues#Twenty.
'                    documentation ```vb\nEnumWithIntValues.Twenty = 20\n```
          End Enum

          Enum EnumWithByteValues
'              ^^^^^^^^^^^^^^^^^^ definition scip-dotnet nuget . . VBMain/Enums#EnumWithByteValues#
'                                 documentation ```vb\nEnum EnumWithByteValues\n```
              Five = &H5
'             ^^^^ definition scip-dotnet nuget . . VBMain/Enums#EnumWithByteValues#Five.
'                  documentation ```vb\nEnumWithByteValues.Five = 5\n```
              Fifteen = &HF
'             ^^^^^^^ definition scip-dotnet nuget . . VBMain/Enums#EnumWithByteValues#Fifteen.
'                     documentation ```vb\nEnumWithByteValues.Fifteen = 15\n```
          End Enum
      End Class
  End Namespace
